### PR TITLE
Bundle Server on PyPi

### LINKED ISSUE #11: Configure server for port 80
The Picobrew machines expect all traffic to occur over the standard HTTP port (`80`). Right now, the server is setup to use a development port (`5000`).  This should be configurable.

### PR DESCRIPTION
**Pull request by hotzenklotz**
In order to make installation and maintenance easier, I added PyPi packaging to the project.

### Changes:
- restructure the project structure to be more Pythonic (moved all files into directory `picobrew_server``
- added Poetry for package managment, bunding and publishing
- added automatic publishing to PyPi with GitHub actions
- update readme
- changed server startup to use the [Flask CLI](https://flask.palletsprojects.com/en/1.1.x/cli/) for easier port configuration. fixes #11 

### Installation steps

1. Install Python 3.7
2. In a terminal download, install and run the project:
```bash
// Download and install
pip install picobrew_server

// Start the server in production mode on port 80
FLASK_APP=picobrew_server flask run --port 80 --host 0.0.0.0
```

### CODE REVIEW

**Comment by hotzenklotz** [comment]
@danyou @fotoapparat @TomKriek99 Does anyone mind testing, if you are able to install and run the server using instructions above. Hopefully using Python packages the installation should be more straightforward.

Thanks for the feedback.

File: .github/workflows/publish_pypi.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Publish to PyPI
```
Comment (hez) [inline .github/workflows/publish_pypi.yml:1]: 👍 This is awesome!

**Comment by danyou** [comment]
I was able to install with Windows, with Administrator cmd.
To run I needed to use:
set FLASK_APP=picobrew_server
flask run --port 80 --host 0.0.0.0
(I did not need the Administrator cmd)

[flask.txt](https://github.com/hotzenklotz/picobrew-server/files/4582841/flask.txt)

**Comment by hotzenklotz** [comment]
@danyou Thanks for double-checking. I updated the README accordingly for Windows users.

**Pull request merged**